packages/rum/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -193,6 +214,9 @@ function newView(
         setLoadEvent(newTimings.loadEvent)
       }
     },
+    addTiming(name: string, time: number) {
+      customTimings[name] = msToNs(time - startTime)
```

Comment:
It should be better to be consistent for where we convert timings to ns.
For other timings, it is done in viewCollection since it is here that we convert internal view representation to raw view event (intake format).
It allow us to internally work with convenient format/unit without depending on intake format.

Reply:
Makes sense, I will update this.